package.json

```diff
@@ -23,6 +23,7 @@
     "start:proxy": "NODE_ENV=development PROXY=true webpack serve --config config/dev.webpack.config.js",
     "start:proxy:beta": "BETA=true NODE_ENV=development PROXY=true webpack serve --config config/dev.webpack.config.js",
     "test": "jest --verbose --passWithNoTests --env=jsdom",
```

Comment:
I think you'd want to nest `test:ct` into `test` command, so that the cypress tests are executed with each CI check